tox.ini

```diff
@@ -155,6 +155,7 @@ deps =
 # test dependencies installed in all envs
     mock
 # used to test our custom msgpack encoder
+    tracer: msgpack
```

Comment:
```suggestion
```

Reply:
I added this line because when I run `tox -e py37-tracer` from a test runner image I get:

```ModuleNotFoundError: No module named 'msgpack'```